apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/VolatileResourceManager.cs

```diff
@@ -137,25 +137,31 @@ public void SinglePhaseCommit(SinglePhaseEnlistment singlePhaseEnlistment)
         {
             var transaction = await GetTransactionAsync(cancellationToken).ConfigureAwait(false);
             if (transaction == null)
+            {
                 throw new InvalidOperationException("Unable to obtain a spanner transaction to execute within.");
+            }
             return await ((ISpannerTransaction) transaction).ExecuteMutationsAsync(mutations, cancellationToken);
```

Comment:
hey jon, does your plugin only run on .net core?  it missed these (and nice catch evil).